### DISCUSSION TITLE
Easy start after installation

### DISCUSSION THREAD

**Discussion by TrueFawkes** [azuritecoin/OnnxDiffusersUI#47] [Ideas]
Love what you are doing.
For a slightly better experience I've created a simple batch file that starts the program a bit easier.
Especially since the memory seems to fill up and you need to restart it again in the current version.

Start-UI.bat in the root of the project.

```
@echo off
echo please wait..
call virtualenv\Scripts\activate.bat
python onnxUI.py
pause
```

Start-UI-cpu.bat
```
@echo off
echo please wait..
call virtualenv\Scripts\activate.bat
python onnxUI.py --cpu-only
pause
```

Also running on a 5800X with 16GB seems much faster than on a 8GB RX580.

**Comment by azuritecoin** [comment]
One of the things I've debated on was if I should activate the virtual envrionment and start the onnxUI within `setup.bat`. I felt it would be better if the user can do some of the operations manually eg. running `setup.bat` then running the conversion scripts.
